css/style.css

```diff
@@ -16,4 +15,149 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+/*HEADER STYLES*/
+
+#karma-logo {
+  width: 25px;
+  margin-left: 50px;
+  margin-top: 50px;
+  margin-bottom: 20px;
```

Comment:
if you want to make this 1 line instead of 3 you can use the shorthand:
margin: 50px 0 20px 50px;
[top right bottom left]